ApiDoctor.Validation/OData/Property.cs

```diff
@@ -47,15 +46,45 @@ public Property()
         [XmlAttribute("Type"), ContainsType, MergePolicy(MergePolicy.PreferLesserValue)]
         public string Type { get; set; }
 
-        [XmlAttribute("Nullable"), MergePolicy(MergePolicy.PreferTrueValue)]
-        public bool Nullable { get; set; }
+        [XmlIgnore]
+        private bool _nullable;
+
+        [XmlAttribute("Nullable"), MergePolicy(MergePolicy.PreferFalseValue)]
+        public bool Nullable {
+            get { return _nullable; }
+            set
+            {
+                _nullable = value;
+                NullableSpecified = true;
+            }
+        }
+
+        [XmlIgnore]
+        public bool NullableSpecified{ get; set;}
 
         [XmlIgnore]
-        public bool NullableSpecified => this.Nullable;
+        private bool _isUnicode;
```

Comment:
same as above. can't we just suppress the attribute when it's false?

Reply:
I think you mean to suppress when the value is true as that is the default.
docs.oasis-open.org/odata/odata-csdl-xml/v4.01/cs01/odata-csdl-xml-v4.01-cs01.html#sec_Nullable
docs.oasis-open.org/odata/odata-csdl-xml/v4.01/cs01/odata-csdl-xml-v4.01-cs01.html#sec_Unicode

The main reason why I don't want to suppress the attributes when they are the default value is to support debugging our tools. Maintaining fidelity with input metadata helps us get a readable/scannable diff. We do it a lot. We don't see a lot of default values being set in attributes.

Reply:
True is the default so we don't want to suppress false.